crates/uv-cli/src/lib.rs

```diff
@@ -4721,6 +4721,16 @@ pub enum PythonCommand {
     /// See `uv help python` to view supported request formats.
     Install(PythonInstallArgs),
 
+    /// Upgrade Python to the latest patch version.
+    ///
+    /// Multiple Python minor versions may be requested. If none are provided, upgrades will be
+    /// attempted for all managed, installed CPython minor versions.
+    ///
+    /// Virtual environments created by uv will transparently upgrade their patch version in the
+    /// case of an upgrade. But if they were created before the upgrade feature was added to uv,
+    /// they would need to be recreated to reflect upgrades.
+    Upgrade(PythonUpgradeArgs),
```

Comment:
What happens if there are multiple patch versions for a given minor, e.g., if you do `uv python install 3.13.1 3.13.2 && uv python upgrade 3.13`? Do we always select `3.13.2` for upgrade? Can you document the behavior?

Reply:
When upgrading, we install the latest available patch if it has not already been installed and leave the other patches in place. I can add more documentation.

Reply:
👍 seems important to note that the previous patches will not be removed.